### PR TITLE
First version of my first Microverse project.

### PR DESCRIPTION
**Pull request by Yothu**
My first pull request for my first Microverse project.

Add files:

- index.html
- style/style.css
- .gitignore
- README.md
- MIT.md
- linters.yml for HTML/CSS

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+<html>
+    <head>
+        <meta charset="utf-8">
+	    <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>Hello Microverse</title>
+    </head>
+    <body>
+        <h1>Hello Microverse!</h1>
+    </body>
+</html>
```
Comment (Yothu) [inline index.html]: All looks wholesome to me, Except the end of the line!
```suggestion
</html>

```

Comment (Yothu) [reply index.html]: Oh my goodness! Thanks for the review!

**Review by gabrielyea — CHANGES_REQUESTED** [review]
# Required Changes :recycle: 
Hi @Yothu , 

You have don a great job setting up your first project. :rocket: 
First let me point out:
- Good commit history!
- You are folloing gitflow

Sadly there are some key features missing from your project, don't worry they are easy to fix! :muscle: . See below :arrow_down: 

<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️

- [x] Please use [this template](https://github.com/microverseinc/readme-template) for your README

---

- [x] Fill the README with your own info, let me show you an example:
image

---

- [x] Please add a description to your PR. If you are not sure how let me show you an example:
image

---

- [x] Don't forget to fill the `MIT` license with your own info! (the mit file is also [here](https://github.com/microverseinc/readme-template))


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rustam-tolipov — APPROVED** [review]
## Hi @Yothu  👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

sheep shon

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**